report/src/system_design/intro.tex

```diff
@@ -1,13 +1,15 @@
-In Chapter \label{sec:presence_intro} technologies that can be utilized for presence detection were described. 
+\section{Introduction: System Design}
+In Chapter \ref{chap:presence_intro} technologies that can be used for presence detection were described. 
 Technology selection criteria based on \citeauthor{presence_ble_review}~\cite{presence_ble_review} were developed and identified Bluetooth as the most suitable technology for a presence system satisfying the user problems described in Section \ref{sec:understanding_the_problem}.
 
-Chapter \ref{sec:presence_detection_using_ble} then details two approaches to using Bluetooth for presence detection. We found that a fingerprinting approach using kNN is suitable for occupancy detection. 
-We argue that the approach can easily be modified with additional labels when collecting data in the first phase of the approach, as described in Secton \ref{sec:fingerprinting}.
+Chapter \ref{sec:presence_detection_using_ble} then details two approaches to using Bluetooth for presence detection. 
+We found that a fingerprinting approach using kNN is suitable for occupancy detection. 
+We argue that the approach can easily be modified with additional labels when collecting data in the first phase of the approach, as described in section \ref{sec:fingerprinting}.
 
-The following sections details how a presence system for room occupancy detection using fingerprinting can be designed and implemented. 
-Section \ref{sec:first_phase} details how we design an application for collecting measurments for the first phase. 
-We describe how we approach the data collection, and how  we accomodate the need for labelling of the collected data.
-Section \ref{sec:ble_implementation} describe important implementation details this application. 
+In the following sections we will first present a general overview of the application and how it supports the two phases of fingerprinting. 
+Section \ref{sec:first_phase} details how we design an application for collecting measurements for the first phase. 
```

Comment:
```suggestion
Section \ref{sec:first_phase} establishes how we approach the collection measurements for the first phase, followed by how this can be done using an App for smartphone. 
```